mlflow/data/huggingface_dataset.py

```diff
@@ -69,25 +69,17 @@ def _compute_digest(self) -> str:
         )
         return compute_pandas_digest(df)
 
-    def _to_dict(self, base_dict: Dict[str, str]) -> Dict[str, str]:
-        """
-        Args:
-            base_dict: A string dictionary of base information about the
-                dataset, including: name, digest, source, and source
-                type.
-
-        Returns:
-            A string dictionary containing the following fields: name,
-            digest, source, source type, schema (optional), profile
-            (optional).
-        """
-        return {
-            **base_dict,
-            "schema": json.dumps({"mlflow_colspec": self.schema.to_dict()})
-            if self.schema
-            else None,
-            "profile": json.dumps(self.profile),
-        }
+    def to_dict(self) -> Dict[str, str]:
+        """Create config dictionary for the dataset."""
```

Comment:
can we keep the relevant parts of the original docstring? this comment applies to various changes like this one